enterprise/admin/views.py

```diff
@@ -62,7 +62,7 @@ def _build_context(self, request, customer_uuid):
         """
         # TODO: pylint acts stupid - find a way around it without suppressing
         enterprise_customer = EnterpriseCustomer.objects.get(uuid=customer_uuid)  # pylint: disable=no-member
-        linked_learners = EnterpriseCustomerUser.objects.filter(enterprise_customer__uuid=enterprise_customer.uuid)
+        linked_learners = EnterpriseCustomerUser.objects.filter(enterprise_customer=enterprise_customer)
```

Comment:
Just out of curiosity – does this change make any difference?  The new code is more concise, and might even result in more efficient SQL, but it should do exactly the same thing, right?